student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] Kindly initialize Student's age, name, and parent_permission as well...
also `super()` mustn't be empty you need to add (age, name, parent_permission) in it.